lms/static/scripts/frontend_apps/components/ErrorDisplay.tsx

```diff
@@ -112,8 +116,8 @@ export default function ErrorDisplay({
   const message = formatErrorMessage(error, /* prefix */ description);
 
   return (
-    <Scrollbox classes="LMS-Scrollbox">
-      <div className="pt-4 space-y-4">
+    <Scroll classes="!mt-0">
+      <ScrollContent classes="pt-4 space-y-4 !px-0" >
```

Comment:
The `ScrollContent` comes with `px-3` by default. The only way to overwrite that and make it look like before, is by overwriting it with `!important` via `!p-0`.

Not very nice though. I'm open to suggestions here.